charts/istio-authorizer/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: istio-authorizer
 description: A Helm chart for istio-authorizer
 type: application
-version: 1.17.5-1
+version: 1.17.6
```

Comment:
If this is not a stable release, should it not point to 1.17.5-2? 
By stable, I mean that app version is pointing to a not yet released authorizer version.